sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py

```diff
@@ -84,6 +78,18 @@
 from apache_beam.options.pipeline_options import PipelineOptions
 from apache_beam.options.pipeline_options import SetupOptions
 
+# Protect against environments where datastore library is not available.
+# pylint: disable=wrong-import-order, wrong-import-position
+try:
+  from google.cloud.proto.datastore.v1 import entity_pb2
+  from google.cloud.proto.datastore.v1 import query_pb2
+  from googledatastore import helper as datastore_helper
+  from googledatastore import PropertyFilter
+  from past.builtins import unicode
+except ImportError:
+  pass
```

Comment:
Maybe we should be able to skip the test is we're unable to install Datastore? What happens if Datastore is not available and we run the test?

Reply:
The test will be skipped because --test-pipeline-options is not specified. Adding the try-catch to make sure the code could be interpreted. Otherwise, exceptions will be through when we import the datastore_wordcount into the IT test.

Reply:
Thank you!